WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -21,13 +21,13 @@ struct ItemRowView: View {
                 Text(cartItem.item.name)
                     .foregroundColor(Color.posPrimaryText)
                     .font(Constants.itemNameFont)
-                    .padding(.horizontal, Constants.horizontalElementSpacing)
-                    .padding(.top, Constants.verticalPadding)
+                    .padding(.leading, Constants.horizontalElementSpacing * (scale / 1))
+                    .padding(.top, Constants.verticalPadding * (scale / 1))
                 Text(cartItem.item.formattedPrice)
                     .foregroundColor(Color.posPrimaryText)
                     .font(Constants.itemPriceFont)
-                    .padding(.horizontal, Constants.horizontalElementSpacing)
-                    .padding(.bottom, Constants.verticalPadding)
+                    .padding(.leading, Constants.horizontalElementSpacing * (scale / 1))
+                    .padding(.bottom, Constants.verticalPadding * (scale / 1))
```

Comment:
Question: Why the `/ 1` bit here? It would always return the same scale value without it, so it has no effect.

Reply:
Yes, that's a mistake 😀

Reply:
Made fixes. Thanks! Now Cart padding reduces when scales increase to fit more text. `Reviewer (or Author, in the case of optional code reviews)` remains.

Reply:
> Reviewer (or Author, in the case of optional code reviews) remains.

Oops, done!